cmd/kubeadm/app/cmd/init.go

```diff
@@ -425,6 +411,19 @@ func (i *Init) Run(out io.Writer) error {
 		return fmt.Errorf("error uploading configuration: %v", err)
 	}
 
+	glog.V(1).Infof("[init] creating kubelet configuration configmap")
+	if err := kubeletphase.CreateKubeletConfigMap(i.cfg, client); err != nil {
+		return fmt.Errorf("error creating base kubelet configuration: %v", err)
+	}
+
+	// NOTE: flag "--dynamic-config-dir" should be specified in /etc/systemd/system/kubelet.service.d/10-kubeadm.conf
```

Comment:
b/c the kubelet will write to this location it should be /var/lib/kubelet ...

Reply:
How are you going to upgrade into this?  You will need a systemctl daemon-reload + restart.